client/src/pages/task/TaskPage.tsx

```diff
@@ -181,6 +180,15 @@ const TaskPage: React.FC = () => {
     }
   ]
 
+  if (task.ideEnabled) {
+    // insert online IDE before last element if enabled
+    tabs.splice(-1, 0, {
+      key: '/ide',
+      tab: tab('Online IDE', 'cloud'),
+      disabled: !answer
+    })
+  }
+
```

Comment:
Should follow the same pattern as `testingModeSwitch`, i.e.

```
const ideTab = task.ideEnabled ? [{
      key: '/ide',
      tab: tab('Online IDE', 'cloud'),
      disabled: !answer
    }] : []
```
then spread into `tabs`